libbeat/publisher/pipeline/module.go

```diff
@@ -88,6 +88,9 @@ func Load(
 				"host": common.MapStr{
 					"name": name,
 				},
+				"ecs": common.MapStr{
+					"version": "1.0.0-beta2",
```

Comment:
Just a thought, if we vendor the generated ECS Go package we could rely on that package’s constant (`ecs.Version`) for keep this value in sync. If you update the fields data you should also be updating the vendored ecs package.

Reply:
Interesting idea. To take this one step further would be nice to have a `fields.go` as part of this package that we could use too so no need to modify the ecs-fields.yml. This would assume that the global fields.yml is generated by `export fields` instead of collection in the future.